decisions/World Bank.txt

```diff
@@ -1051,3 +1052,17 @@ political_decisions = {
 		}
 	}
 }
+
+# Tech Clean Up
+political_decisions = {
+	MOR_expansion_5_if_ring = {
+		potential = {
+			tag = FOR
+			MOR = { NOT = { expansion_5 = 1 } }
+		}
+		
+		allow = { MOR = { has_country_flag = has_one_ring } }
+		
+		effect = { MOR = { activate_technology = expansion_5 } }
```

Comment:
Probably should give Mordor the whole line if they don't have it. I'd either also make this only for ai Mordor, or call an event that gives Mordor the technology -- imagine a player's surprise when all of a sudden they get a tech for free with no notice. Probably wouldn't even notice, or just be confused

Reply:
Ok. I'll make an event that will fire if those conditions are met.

Reply:
Made the event, pushed the commit.